features/netsocket/emac-drivers/TARGET_STM/stm32xx_emac.cpp

```diff
@@ -279,10 +294,10 @@ bool STM32_EMAC::low_level_init_successful()
     /* Init ETH */
     uint8_t MACAddr[6];
     EthHandle.Instance = ETH;
-    EthHandle.Init.AutoNegotiation = ETH_AUTONEGOTIATION_ENABLE;
+    EthHandle.Init.AutoNegotiation = MBED_CONF_STM32_EMAC_ETH_PHY_AUTONEGOCIATION;
     EthHandle.Init.Speed = ETH_SPEED_100M;
     EthHandle.Init.DuplexMode = ETH_MODE_FULLDUPLEX;
```

Comment:
Why not make this configurable too?

Reply:
If you think it can be useful, I can add this parameter ?

Reply:
Yes, that would be useful for us :)